ggdeploymentd/src/bootstrap_manager.c

```diff
@@ -111,6 +111,26 @@ GglError save_iot_jobs_id(GglBuffer jobs_id) {
     return GGL_ERR_OK;
 }
 
+GglError save_iot_jobs_version(int64_t jobs_version) {
+    GGL_LOGD("Saving IoT Jobs version %li in case of bootstrap.", jobs_version);
```

Comment:
Use the fixed-width macros for printing int64_t (build fails with warning arm32)
```suggestion
    GGL_LOGD("Saving IoT Jobs version %" PRIi64 " in case of bootstrap.", jobs_version);
```

Reply:
add `#include <inttypes.h>` to the include list for the macro